### PR TITLE
Update RAiDER-Python tutorial

### PR DESCRIPTION
**Pull request by jlmaurer**
- Updated raider tutorial in Python to run both an example with HRRR and HRRR-AK

### CODE REVIEW

**Comment by jlmaurer** [comment]
This can be merged after merging [#507 in RAiDER](https://github.com/dbekaert/RAiDER/pull/507)

**Review by dbekaert — APPROVED** [review]
LGTM @jlmaurer

**Pull request merged**
